source/guidelines/sdk-development.rst

```diff
@@ -64,215 +68,142 @@ Reactive
-        "AliasDTO": AliasDTO,
-        "AnnounceTransactionInfoDTO": AnnounceTransactionInfoDTO,
-        "BlockDTO": BlockDTO,
-        "BlockInfoDTO": BlockInfoDTO,
-        "BlockMetaDTO": BlockMetaDTO,
-        "BlockchainScoreDTO": BlockchainScoreDTO,
-        "CommunicationTimestamps": CommunicationTimestamps,
-        "Cosignature": Cosignature,
-        "HeightInfoDTO": HeightInfoDTO,
-        "MerklePathItem": MerklePathItem,
-        "MerkleProofInfo": MerkleProofInfo,
-        "MerkleProofInfoDTO": MerkleProofInfoDTO,
-        "MosaicDTO": MosaicDTO,
-        "MosaicDefinitionDTO": MosaicDefinitionDTO,
-        "MosaicIds": MosaicIds,
-        "MosaicInfoDTO": MosaicInfoDTO,
-        "MosaicMetaDTO": MosaicMetaDTO,
-        "MosaicNamesDTO": MosaicNamesDTO,
-        "MosaicPropertyDTO": MosaicPropertyDTO,
-        "MultisigAccountGraphInfoDTO": MultisigAccountGraphInfoDTO,
-        "MultisigAccountInfoDTO": MultisigAccountInfoDTO,
-        "MultisigDTO": MultisigDTO,
-        "NamespaceDTO": NamespaceDTO,
-        "NamespaceIds": NamespaceIds,
-        "NamespaceInfoDTO": NamespaceInfoDTO,
-        "NamespaceMetaDTO": NamespaceMetaDTO,
-        "NamespaceNameDTO": NamespaceNameDTO,
-        "NetworkTypeDTO": NetworkTypeDTO,
-        "NodeInfoDTO": NodeInfoDTO,
-        "NodeTimeDTO": NodeTimeDTO,
-        "ResolutionEntryDTO": ResolutionEntryDTO,
-        "ResolutionStatementDTO": ResolutionStatementDTO,
-        "ServerDTO": ServerDTO,
-        "ServerInfoDTO": ServerInfoDTO,
-        "SourceDTO": SourceDTO,
-        "StatementsDTO": StatementsDTO,
-        "StorageInfoDTO": StorageInfoDTO,
-        "TransactionHashes": TransactionHashes,
-        "TransactionIds": TransactionIds,
-        "TransactionInfoDTO": TransactionInfoDTO,
-        "TransactionMetaDTO": TransactionMetaDTO,
-        "TransactionPayload": TransactionPayload,
-        "TransactionStatementDTO": TransactionStatementDTO,
-        "TransactionStatusDTO": TransactionStatusDTO,
-    }
-
-5. Copy the generated files into the `nem2-sdk infrastructure folder <https://github.com/nemtech/nem2-sdk-typescript-javascript/tree/master/src/infrastructure>`_.
-
-6. Drop the generated client classes and implement them using the `Repository pattern <https://martinfowler.com/eaaCatalog/repository.html>`_ returning `Observables <https://en.wikipedia.org/wiki/Observer_pattern>`_ of `ReactiveX <http://reactivex.io/>`_.
-
-Example of a Repository and HTTP implementation:
+    openapi-generator generate -i ./openapi3.yml -g typescript-node -o ./symbol-ts-sdk/ && rm -R symbol-ts-sdk/test
+
+4. The generated lib are normally published into a central repository (e.g. maven, npm). The SDKs depend on those libraries like any other third party dependency. To automate the deployment of the packages, including the generator for the selected programming language in the `symbol-openapi-generator <https://github.com/nemtech/symbol-openapi-generator>`_ project.
```

Comment:
Should we mention that Travis is automatically upgrading/releasing new versions of the generated clients when the open api spec is updated? Currently, one of us updates the open api, everybody gets the new clients deployed into the repos. The creator of the new SDK should follow the same pattern and get the same benefits. We can then send a Slack ping saying that there is a new version of the open api clients, dev should update their sdks